pkg/descheduler/descheduler_test.go

```diff
@@ -74,7 +91,9 @@ func TestTaintsUpdated(t *testing.T) {
 	var evictedPods []string
 	client.PrependReactor("create", "pods", podEvictionReactionFuc(&evictedPods))
 
-	internalDeschedulerPolicy, err := v1alpha1.V1alpha1ToInternal(client, dp, pluginregistry.PluginRegistry)
+	internalDeschedulerPolicy := &api.DeschedulerPolicy{}
+	scope := scope{}
+	err = v1alpha1.V1alpha1ToInternal(dp, pluginregistry.PluginRegistry, internalDeschedulerPolicy, scope)
```

Comment:
Other option is to make dp already be declared in our internal type so the test doesn't have to convert it (since the test is testing something else).

But I guess it is also nice we have the conversion being called at some other places so it is validates in other contexts.

Optional change if you think it makes sense